src/actions.ts

```diff
@@ -341,16 +341,23 @@ export const goToLineBoundary = (
 export const navigateLine = (
   editor: Editor,
   selection: EditorSelection,
-  direction: 'up' | 'down',
+  position: 'next' | 'prev' | 'top' | 'bottom',
```

Comment:
_nit_: `first` and `last` might be slightly clearer than `top` and `bottom` (to match the command name)

Reply:
Yes, I agree with that. I'll fix it.

Reply:
Fixed